bridges/PicukiBridge.php

```diff
@@ -0,0 +1,92 @@
+<?php
+class PicukiBridge extends BridgeAbstract
+{
+
+	const MAINTAINER = 'marcus-at-localhost';
+	const NAME = 'Picuki Bridge';
+	const URI = 'https://www.picuki.com/';
+	const CACHE_TIMEOUT = 21600; // 6h
+	const DESCRIPTION = 'Returns Picuki posts by users and by hashtag';
+
+	const PARAMETERS = [
+		'Username' => [
+			'u' => [
+				'name' => 'username',
+				'required' => true,
+			],
+		],
+		'Hashtag' => [
+			'h' => [
+				'name' => 'hashtag',
+				'required' => true,
+			],
+		]
+	];
+
+	public function getURI()
+	{
+		if (!is_null($this->getInput('u'))) {
+			return trim(self::URI, '/') . '/profile/' . $this->getInput('u');
+		}
+
+		if (!is_null($this->getInput('h'))) {
+			return trim(self::URI, '/') . '/tag/' . $this->getInput('h');
```

Comment:
That would be nice, if `example_hash_tag` is accepted the same like `#example_hash_tag`